modules/dkan_common/drush.services.yml

```diff
@@ -1,13 +1,13 @@
 services:
```

Comment:
Let's make a follow-up issue to contemplate modernizing the command files so they don't need a services file. https://www.drush.org/11.x/dependency-injection/#create-method